ktor-server/ktor-server-netty/jvm/src/io/ktor/server/netty/NettyChannelInitializer.kt

```diff
@@ -36,7 +36,8 @@ public class NettyChannelInitializer(
     private val runningLimit: Int,
     private val responseWriteTimeout: Int,
     private val requestReadTimeout: Int,
-    private val httpServerCodec: () -> HttpServerCodec
+    private val httpServerCodec: () -> HttpServerCodec,
+    private val channelPipelineConfig: ChannelPipeline.() -> Unit
```

Comment:
I would prefer to add a fallback constructor for those who call it directly for some reason.

Reply:
Instead of having another constructor, I feel adding a default value to the parameter is more than sufficient.  Any comment?
`private val channelPipelineConfig: ChannelPipeline.() -> Unit = {}`

Reply:
added a default parameter to the constructor.

Reply:
Unfortunately, it doesn't help because the binary compatibility is broken here. We need a secondary constructor with the old signature. This is because of our compatibility guarantees. After doing this run `./gradlew apiDump` and ensure that we only add new signatures and don't remove/change anything.

Reply:
@cy6erGn0m , sorry for the delayed response. I will fix it during this week